lib/ssm.ts

```diff
@@ -0,0 +1,51 @@
+import AWS from 'aws-sdk';
+
+const SSM = new AWS.SSM();
+
+interface ParameterValues {
+    [key: string]: string;
+}
+
+export const getParemetersFromSsm = async (
+    path: string, // Path of SSM paramters
+    parameterNames: string[] // Name within the specified path
+) => {
```

Comment:
I would prefer to generic this function and let it return a typed object so we know exactly what we can use instead of a `[key: string]: string` object.